### PR TITLE
JHy py scripts update

### PR DESCRIPTION
**Pull request by janhybs**
Add python interpret wrapper

### CODE REVIEW

File: bin/python/runtest.py [hunk 1]
```diff
@@ -83,12 +82,14 @@
...
+    from scripts.core.threads import BinExecutor
     from scripts.runtest_module import do_work
-
-    # for debug only set dir to where script should be
+    #
+    # # for debug only set dir to where script should be
     Paths.base_dir(__file__)
-    # Paths.base_dir('/home/jan-hybs/Dokumenty/projects/Flow123d/flow123d/bin/python/foo')
-
+    Paths.base_dir('/home/jan-hybs/Dokumenty/projects/Flow123d/flow123d/bin/python/foo')
```
Comment (jstebel) [inline bin/python/runtest.py]: Should be removed (prevents finding the binaries flow123d, mpiexec, ndiff).

Comment (x3mSpeedy) [inline bin/python/runtest.py]: My bad, commented for now, since I will definitely use it later on.

**Comment by jstebel** [comment]
Some comments to runtest.py:
- YAML test files should be run in the order given in config.yaml.
- If reference output directory does not exist then the test should fail.
- Error status was sometimes wrong (random behaviour, cannot reproduce).
- Log file run-test-...log should be placed in the directory of the YAML file.

**Comment by x3mSpeedy** [comment]
Additional comments:

- [ ] implement `runtest.py` output format on demand (perhaps `json` format)
- [ ] edit `runtest.py`  in `-q` mode to call `runtest.py` in local mode in PBS script file, communication via output `json` file

File: tests/03_transport_small_12d/config.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+common_config:
+  proc: [1]
+  time_limit: 30
+  memory_limit: 400
+  check_rules:
+      - ndiff:
+          files: ["*"]
```
Comment (jstebel) [inline tests/03_transport_small_12d/config.yaml]: Make the comparison of all files by ndiff default setting.

File: tests/03_transport_small_12d/config.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# Main list contain one item for every active test.
+test_cases:
+  - file: [flow_implicit.yaml]
```
Comment (jstebel) [inline tests/03_transport_small_12d/config.yaml]: In case of single file allow input without brackets.

**Comment by jbrezmorf** [comment]
Base branch changed to master.
Seems that part of these changes is still out of master.
If these changes should be merged into master, please, resolve conflicts first, then if it pass CI we can merge back to master. Otherwise delete the branch.

**Comment by janhybs** [comment]
These changes were merged to a different branch about 2 years ago. They are now in master branch as well. This branch can be deleted.

**Pull request closed without merging**